### PR TITLE
[OoT] Improvements and fixes for scene exporter panel

### PR DESCRIPTION
**Pull request by Yanis42**
Changes:
- Added custom path option for decomp mode, the remove operator grays out if the path doesn't exists
- Added a box for HackerOoT export options to make it less confusing

### CODE REVIEW

File: fast64_internal/oot/oot_level_writer.py [hunk 1]
```diff
@@ -1003,63 +1001,103 @@ def draw(self, context):
...
                 prop_split(col, settings, "subFolder", "Subfolder")
                 prop_split(col, settings, "name", "Name")
 
         prop_split(col, context.scene, "ootSceneExportObj", "Scene Object")
 
+        col.prop(settings, "singleFile")
+        col.prop(settings, "customExport")
+
         if context.scene.fast64.oot.hackerFeaturesEnabled:
+            hackerOoTBox = col.box()
```
Comment (kurethedead) [inline fast64_internal/oot/oot_level_writer.py:1034]:
```suggestion
            hackerOoTBox = col.box().column()
```
Don't use the results of box() directly, this causes big spaces between lines.

File: fast64_internal/oot/oot_level_writer.py [hunk 2]
```diff
@@ -1003,63 +1001,103 @@ def draw(self, context):
...
+            exportPath = (
+                context.scene.ootDecompPath + f"assets/scenes/{removeSettings.subFolder}/{removeSettings.name}/"
+            )
+
+        removeRow = col.row()
+        removeRow.operator(OOT_RemoveScene.bl_idname, text="Remove Scene")
+
+        if removeSettings.option == "Custom" and not os.path.exists(exportPath):
+            removeRow.enabled = False
+            col.label(text="This path doesn't exists.")
```
Comment (kurethedead) [inline fast64_internal/oot/oot_level_writer.py:1098]:
```suggestion
            col.label(text="This path doesn't exist.")
```

**Review by kurethedead — CHANGES_REQUESTED** [review]
I'm opposed to splitting up the scene operators into separate panels - I did this before with SM64 and it ended up being so overwhelming that the panels were merged together.

**Comment by Yanis42** [comment]
The alternative is using ``col.box()`` to make things easier to read, I can revert these changes to that alternative if that's ok 👍

File: fast64_internal/oot/oot_level_writer.py (hunk 1 above)
Comment (Yanis42) [reply fast64_internal/oot/oot_level_writer.py:1034]: good to know

**Review by kurethedead — APPROVED** [review]
Seems good to me, will merge unless there any other objections.

**Pull request merged**